### PR TITLE
Security/np create- Network Policy Modification

### PR DESCRIPTION
**Pull request by venkat-99**

### CODE REVIEW

File: deployment/kubernetes/helm/charts/authenticator-service/templates/networkpolicy.yaml
Comment (sathishkumar-p) [inline deployment/kubernetes/helm/charts/authenticator-service/templates/networkpolicy.yaml:1]: can we check kong is connecting auth service ?

File: deployment/kubernetes/helm/charts/storage-be/templates/networkpolicy.yaml [hunk 1]
```diff
@@ -4,9 +4,22 @@ metadata:
   name: {{ include "storage-be.fullname" . }}
   namespace: {{ .Values.namespace }}
 spec:
-  ingress:
-    - ports:
-        - port: {{ .Values.containerPort }}
   podSelector:
     matchLabels:
```
Comment (sathishkumar-p) [inline deployment/kubernetes/helm/charts/storage-be/templates/networkpolicy.yaml:8]: kong rule is missing , please check for other services

File: deployment/kubernetes/helm/charts/airflow/templates/networkpolicy.yaml [hunk 2]
```diff
@@ -9,8 +9,17 @@ spec:
       name: {{ include "airflow-service.name" . }}
   ingress:
     - from:
-      - podSelector:
```
Comment (sathishkumar-p) [inline deployment/kubernetes/helm/charts/airflow/templates/networkpolicy.yaml:12]: airflow backend service np is changed ?

Comment (VardhanDeva) [inline deployment/kubernetes/helm/charts/airflow/templates/networkpolicy.yaml:12]: dummy comment

**Pull request closed without merging**